scss/themes/tippy/themes/_gorila-gray.scss

```diff
@@ -0,0 +1,54 @@
+@import '../mixins';
+@import '../variables';
+
+$gorilaGrayBg: $base-background;
+$gorilaFontColor: $extra-dark-gray;
```

Comment:
Please remove these and update the files to use colors from our color files

Reply:
Do you mean use the variables $base-background and $extra-dark-gray directly?

Reply:
And if there's any color that is here but not in the colors file, add it there too